File: frontend/src/types/shippingPlan.ts
export interface IShippingPlan {
	id: string | null
	date?: string
	numberOfOrders: number | null
	sumMoney: number | null
	quantity: number | null
}

export interface IShippingPlanDTO {
	id: string
	date: number
	numberOfOrders: number
	sumMoney: string
	quantity: number
}

File: backend/internal/models/shipping_plan.go
package models

type ShippingPlan struct {
	Id             string `json:"id" db:"id"`
	Date           int64  `json:"date" db:"date" binding:"required"`
	NumberOfOrders int    `json:"numberOfOrders" db:"number_of_orders"`
	SumMoney       string `json:"sumMoney" db:"sum_money"`
	Quantity       int    `json:"quantity" db:"quantity"`
}
